openslides/core/static/js/core/base.js

```diff
@@ -689,7 +689,7 @@ angular.module('OpenSlidesApp.core', [
  * one of the ids in filterArray. id_function could also return a list of ids. Example:
  * Item 1 has two tags with ids [1, 4]. filterArray == [3, 4] --> match
  */
-.filter('SelectMultipleFilter', [
+.filter('MultiselectFilter', [
```

Comment:
Why this filter is in base.js and not in site.js? Does we need this filter also on projector?